### ISSUE TITLE
Question about .lib: OIS.lib and OIS_d.lib

### ISSUE THREAD

**Issue by professor-oats** [wgois/OIS#87]
Hello!

Thank you very much for this repository. I am going forward with some tutorials for OGRE and situation has it that I would like to link libraries from OIS into the project.

I am building with CMAKE and make sure to export all files (resolve all symbols for Win) and still can't find the .lib files.

I am in the start of this journey so if you have guidance on how to get .lib I will be glad of your help. I will attach a print of the errorlist when building the OIS through MCVS.

![errorlist](https://github.com/wgois/OIS/assets/75143067/60928d57-ceab-4281-8752-d3fda27ac5b4)

**Comment by Ybalrid** [comment]
Hi, you need to build the OIS source code in Visual studio too, in both Release and Debug configurations

**Comment by Ybalrid** [comment]
Tho I am curious what tutorial are you following. I would guess modern OGRE tutorials might just use SDL2 instead of OIS?

**Comment by professor-oats** [comment]
Hello! Yes, you are perfectly right. I built it in the correct way now and made sure to do it against each configurations. It works.

Also here you are right - it's an old book I found which is a good complement for the rest of the sources and the modern versions use SDL2.

Now I have options and I may do SDL2 when following the book as well.

Your comment made me realise where I was wrong building.

Thanks!
